nikola/plugins/task/galleries.py

```diff
@@ -725,6 +725,9 @@ def gallery_rss(self, img_list, dest_img_list, img_titles, lang, permalink, outp
         def make_url(url):
             return urljoin(self.site.config['BASE_URL'], url.lstrip('/'))
 
+        # WebP is not yet an official MIME type but we need to recognize them
+        mimetypes.add_type('image/webp', '.webp')
```

Comment:
I’m not sure if this is the best placce for this, considering the enclosure feature may also trigger an exception. Perhaps a better place would be somewhere in `nikola.py`, in `Nikola.__init__` so that the mimetype is always defined, even if we are not rendering a gallery RSS file.

Reply:
Hi. I can do this. Am AFK now but will try later.

Reply:
This is done.